GameData/RP-1/Contracts/RP0_Contract_Resources.cfg

```diff
@@ -66,6 +66,32 @@ RESOURCE_DEFINITION
 	}
 }
 
+// =================================================================
+// Resources related to the station science mechanics
+// =================================================================
+
+RESOURCE_DEFINITION
+{
+  name = ResearchPayload
+  density = 0.001
+  unitCost = 2.0
+  flowMode = ALL_VESSEL
+  transfer = PUMP
+  isTweakable = true
+  volume = 1
+}
+
+RESOURCE_DEFINITION
+{
+  name = ProcessedPayload
```

Comment:
Cant see this used anywhere. Remove?

Reply:
Are you specifically referring to the only ProcessedPayload part, or both Payload types? Because ResearchPayload is something that is used, but maybe you mean it doesn't need to be defined here?